recipe/conda_build_config.yaml

```diff
@@ -1,4 +1,4 @@
 channel_sources:
   - conda-forge/label/rust_dev,conda-forge
-MACOSX_SDK_VERSION:        # [osx and x86_64]
-  - "10.12"                # [osx and x86_64]
+# MACOSX_SDK_VERSION:        # [osx and x86_64]
```

Comment:
You should remove this, this is the default now.

Reply:
Thanks, done.